src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -1353,4 +1352,46 @@ private AdsDataProvider() {
                 }
         };
     }
+
+  @DataProvider
+  public static Object[][] adsVelesTracking() {
+    final String TURN_ON_VELES = "?InstantGlobals.wgAdDriverVelesBidderCountries=[XX]";
+
+    return new Object[][]{
+        {
+            new Page(WIKI_SPECIAL, "SyntheticTests/RTB/Prebid.js/Veles/Both/Leaderboard" + TURN_ON_VELES),
+            ImmutableMap.builder()
+              .put(AdsContent.TOP_LB, "20.00")
+              .put(AdsContent.INCONTENT_PLAYER, "USED")
+              .build()
+        },
+        {
+            new Page(WIKI_SPECIAL, "SyntheticTests/RTB/Prebid.js/Veles" + TURN_ON_VELES),
+            ImmutableMap.builder()
+                .put(AdsContent.TOP_LB, "20.00")
+                .put(AdsContent.INCONTENT_PLAYER, "20.00")
+                .build()
+        },
+        {
+            new Page(WIKI_SPECIAL, "Project43_Wikia" + TURN_ON_VELES), // Veles Timeout (page without VAST)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)